### PR TITLE
Save for later num articles

### PR DESCRIPTION
**Pull request by NathanielBennett**
Puts the number of saved articles in the profile dropdown: 

num-articles

### CODE REVIEW

**Comment by robertberry** [comment]
Should there not be a space between 'Saved articles' and '(3)'?

File: static/src/javascripts/projects/common/modules/loyalty/save-for-later.js [hunk 1]
```diff
@@ -1,22 +1,32 @@
...
+    qwery,
+
+     _,
     config,
     mediator,
+
     identity
+
 ) {
+    var $ = function $(selector, context) {
```
Comment (robertberry) [inline static/src/javascripts/projects/common/modules/loyalty/save-for-later.js]: This already exists in common/utils I think

File: static/src/javascripts/projects/common/modules/loyalty/save-for-later.js [hunk 2]
```diff
@@ -57,10 +67,17 @@ define([
...
                 }
+                self.updateNumArticles();
             }
         );
     };
 
+    SaveForLater.prototype.updateNumArticles = function () {
+        var self = this,
+            saveForLaterProfileLink = $('.brand-bar__item--saved-for-later');
+        saveForLaterProfileLink.text('Saved articles(' + self.userData.articles.length + ')');
```
Comment (robertberry) [inline static/src/javascripts/projects/common/modules/loyalty/save-for-later.js]: `articles (`

**Comment by robertberry** [comment]
:+1: otherwise

**Comment by robertberry** [comment]
Ship it or close it.

**Comment by robertberry** [comment]
@NathanielBennett OIIII

**Comment by NathanielBennett** [comment]
I'm closing it. It's going in the next one

**Pull request closed without merging**